src/widgetastic_patternfly/__init__.py

```diff
@@ -1804,3 +1805,150 @@ def items(self):
             # value will include the label from the <strong> block, parse it out
             items.update({key: element_text.replace(key, '', 1).lstrip()})
         return items
+
+
+class DatePicker(Widget):
+    """Represents the Bootstrap DatePicker.
+
+      Args:
+        name: name of DatePicker
+        id: id of DatePicker
+
+    .. code-block:: python
+
+       date = DatePicker(name='miq_date_1')
+    """
+    ROOT = ParametrizedLocator(".//*[contains(@class, 'datepicker-dropdown')]")
+    TEXTBOX = ParametrizedLocator(".//*[@name='{@name}' or @id='{@id}']")
+    DATE = ".//*[contains(@class, 'datepicker-days')]/table/tbody/tr/*"
```

Comment:
REQUIRED: sorry, I probably explained not well enough what I want. All this stuff should be turned into widgets inside main widget. 
This is not the way we develop our widgets. Please look at examples.

Reply:
@izapolsk: I added small changes but I am not able to convert all into widgets. I converted some; which were possible. 

`DATE`, `MONTH`, `YEAR` are `table` but it's difficult and hectic task to operate on the element as per `DatePicker` requirement. Here, we required all `cell` elements of the table but  it have special header elements (`<<`, `>>`, and `Switch`); As per `Table` widget `rows` are mapped with `headers` result, getting extra rows with special characters like `<<` header. It's makes sorting difficult and Implementation will increase the line of code. As, we have to some operation on `classes` like checking `active`, `old` etc status. So, I found best to go with all elements.

Reply:
@digitronik, I'll look over your changes and let's discuss this today.